src/sentry/tempest/tasks.py

```diff
@@ -0,0 +1,178 @@
+import logging
+
+from sentry import http
+from sentry.models.projectkey import ProjectKey, UseCase
+from sentry.tasks.base import instrumented_task
+from sentry.tempest.models import TempestCredentials
+from sentry.utils import json
+
+TEMPEST_URL = ""  # FIXME: Set this value once we have it
+POLL_LIMIT = 348  # 348 every 5 min ~ 100k a day
+
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(
+    name="sentry.tempest.tasks.poll_tempest",
+    queue="tempest",
+)
+def poll_tempest(**kwargs):
+    # FIXME: Once we have more traffic this needs to be done smarter.
+    for credentials in TempestCredentials.objects.all():
+        if credentials.latest_fetched_item_id is None:
+            fetch_latest_item_id.delay(credentials.id)
+        else:
+            poll_tempest_crashes.delay(credentials.id)
+
+
+@instrumented_task(
+    name="sentry.tempest.tasks.fetch_latest_item_id",
+    queue="tempest",
+)
```

Comment:
We should also add at least: 
```
 silo_mode=SiloMode.REGION,
```
keyword arg in `@instrumented_task` to limit to control where will the task be spawned and have silo bound safety for ORM

Reply:
Also, might be good idea to add:
- `soft_time_limit` (in seconds)
- `time_limit` in seconds

To explicitly time limit the task execution